doc/conf.py

```diff
@@ -251,6 +251,10 @@
 
 warnings_filter_config = str(ZEPHYR_BASE / "doc" / "known-warnings.txt")
 
+# -- Options for zephyr.link-roles ----------------------------------------
+
+link_roles_default_url = "https://github.com/zephyrproject-rtos/zephyr"
```

Comment:
I'd name it `link_roles_manifest_baseurl`, this should only be used by the repository containing the manifest.

Reply:
Fixed. Also introduced `link_roles_manifest_project` to get the name of the manifest project.